cnosdb/config.properties

```diff
@@ -0,0 +1,59 @@
+DB_SWITCH=CnosDB
```

Comment:
maybe we can remove this file because it will be generated from the conf directory when packaging?